### ISSUE TITLE
[BUG] Status: 500 Error: chatflowsService.getAllChatflows - read ECONNRESET

### ISSUE THREAD

**Issue by stefanos87** [FlowiseAI/Flowise#3476]
**Describe the bug**
I occasionally receive this message regardless of whether I'm using Flowise through the interface or via the API. Unfortunately, I'm unable to replicate it as it happens sporadically.

**To Reproduce**
I'm unable to reproduce the behavior

**Screenshots**
![Screenshot 2024-11-07 alle 12 13 57](https://github.com/user-attachments/assets/9a0db78b-ec62-4254-8e21-a3043cd16ae5)

**Flow**
If applicable, add exported flow in order to help replicating the problem.

**Setup**

-   Installation docker in swarm mode with traefik 
-   Flowise Version 1.8.4 until 2.1.1
-   OS: Linux
-   Browser chrome

**Comment by HenryHengZJ** [comment]
Seems like the server is not ready yet, give it a bit of time everything should work fine.

**Comment by stefanos87** [comment]
thanks for the reply, all services are up and running flowise works fine, the problem is that this error pops up sporadically, it can happen both from the interface and by querying the api. It seems to be a database problem because if I use mysql the error occurs while using sqlite I did not encounter the error. The thing is, I can't find anything that helps in the logs other than the error I posted.
